### DISCUSSION TITLE
Changing e.g. play button styles with Tailwind CSS

### DISCUSSION THREAD

**Discussion by DamianGlowala** [vidstack/player#1165] [Q&A]
Is it possible to change the styles of e.g. play button with Tailwind CSS? How would a simple example of the markup look like?
![image](https://github.com/vidstack/player/assets/48835293/d3f1cef1-ef03-4824-a25f-1bb1a18527ab)

**Comment by mihar-22** [comment]
You would need to build a custom layout and you can customize whatever you like with Tailwind CSS. The Default Layout can be customized using CSS. See the docs and our examples for getting started with custom layouts and Tailwind.
